src/dodal/plans/verify_undulator_gap.py

```diff
@@ -0,0 +1,70 @@
+from dodal.devices.DCM import DCM
+from dodal.devices.undulator import Undulator, UndulatorGapAccess
+from dodal.log import LOGGER
+
+
+def _get_energy_to_distance_table(lookup_table_path: str) -> dict[float, float]:
+    energy_to_distance_table: dict[float, float] = {}
+    with open(lookup_table_path, mode="r") as table:
+        table_start = False
+        for line in table:
+            line = line.strip()
+            if line.startswith("Units"):
+                table_start = True
+                continue
+            if table_start:
+                value = line.split()
+                energy_to_distance_table[float(value[0])] = float(value[1])
+    return energy_to_distance_table
```

Comment:
Should: I think we can just do the same that we do in `parse_table` in `det_dist_to_beam_converter.py` here, which is:
```python
from numpy import loadtxt
rows = loadtxt(self.lookup_file, delimiter=" ", comments=["#", "Units"])
```
In fact, given this is a common GDA file format can we put it in some util?

Reply:
Det/distance is a simple straight line interpolation. Gap/Energy is more complex and we may need to just sit together and work out how it is going to operate as we switch gap across the energy range.

Reply:
> Should: I think we can just do the same that we do in `parse_table` in `det_dist_to_beam_converter.py` here, which is:
> 
> ```python
> from numpy import loadtxt
> rows = loadtxt(self.lookup_file, delimiter=" ", comments=["#", "Units"])
> ```
> 
> In fact, given this is a common GDA file format can we put it in some util?

I don't think we even need the util function, we can just use `loadtxt` as is